File: challenges/1-produto-simples/java/azuosz/ProdutoSimples.java
import  java.util.Scanner;

public class ProdutoSimples {

    public static void main(String[] args) {

        Scanner scanner = new Scanner(System.in);

        System.out.print("Digite um numero: ");
        int  num1 = (int) scanner.nextInt();

        System.out.print("Digite outro numero: " );
        int num2 = scanner.nextInt();

        int produto = num1*num2;
        System.out.println("Produto: " + produto );





    }

}

File: challenges/1-produto-simples/C/pmba/produto-simples.c
#include <stdio.h>

int main(void)
{
    int num1, num2;
    printf("digite dois numeros: ");
    scanf("%d %d", &num1, &num2);

    int res = num1 * num2;
    printf("produto: %d\n", res);
    return 0;
}
